app/components/pill.tsx

```diff
@@ -15,15 +19,17 @@ const Pill = () => {
     }
   };
 
+  const status = AddressData[name as keyof AddressDataType] || "No data";
+
   return (
     <Box>
       <Text
-        bgColor={getBgColor(AddressData.softStory)}
+        bgColor={getBgColor(status)}
```

Comment:
I liked the separating the hazard data from the colors for the hazard card above. Is it preferred to do it differently here?

Reply:
Thanks for pointing that out! I’ll keep it like this for now, but we can revisit once the updated design is confirmed.